.travis.yml

```diff
@@ -3,7 +3,11 @@ language: java
 jdk:
   - oraclejdk8
 
-install: mvn clean install
+install:
+  - mvn -s .travis.maven.settings.xml -version -B
```

Comment:
I'd gladly say "no problem with -fae" if the whole PR was not outdated due to other PRs being merged faster. Nevertheless, I can say that sadly...